tools/c7n_mailer/README.md

```diff
@@ -158,6 +159,8 @@ Slack integration for the mailer supports several flavors of messaging, listed b
 | Yes       | `slack://#custodian-test` | string      | Send to the Slack channel indicated in string, i.e. #custodian-test |
 | No        | `slack://webhook/#c7n-webhook-test` | string      | **(DEPRECATED)** Send to a Slack webhook; appended with the target channel. **IMPORTANT**: *This requires a `slack_webhook` value defined in the `mailer.yml`.* |
 
+Slack delivery can also be set via a resource tag. For example, using "slack://tag/slack_channel" could be used with a resource tag "slack_channel:https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX" to send to a specified webhook.
```

Comment:
minor could you split this into the resource tag name and tag value, else its a little ambiguous to a reader on whether the value includes that prefix, when it was actually meant to denote the tag key.

Reply:
Probably it should be in the existing table?

| Yes       | `slack://tag/resource-tag-name` | string      | Send to target found in resource tag.  Example of value in tag: `https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX` |

Will it work with any of the formats in the tag value or just the webhook?  If any it can just say "Value of tag may be any of the target formats, e.g. `slack://foo@bar.com`

Reply:
@stefangordon I've only had success sending with webhooks so that is what I've tested - my company blocks legacy tokens and I believe that's what the other methods use? With this in mind, I can re-word this to say explicitly for webhooks. It should take whatever the value is after /tag/ so I would assume it would would work but I just haven't tested.